src/transaction-types.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable no-restricted-globals */
+import { Hex } from './hex';
+
+export type Transaction = (LegacyTx | EIP2930Tx | EIP1559Tx) & Signature;
+
+export type Signature = {
+  /**
+   * EC signature parameter
+   * 32 bytes long sequence.
+   */
+  r?: bigint | string | number | Buffer;
+
+  /**
+   * EC signature parameter
+   * Signature proof.
+   * 32 bytes long sequence
+   */
+  s?: bigint | string | number | Buffer;
+
+  /**
+   * Recovery identifier. It can be either 0x1b or 0x1c
+   * 1 byte long sequence
+   */
+  v?: bigint | string | number | Buffer;
+};
```

Comment:
We could use the `Bytes` type from `bytes.ts` here.

Reply:
[0c59e38](https://github.com/MetaMask/utils/pull/74/commits/0c59e3846a5d9bd90da3b8008170df263ed881c6)